app/views/partials/_navigation.html.haml

```diff
@@ -0,0 +1,17 @@
+%nav.navbar.navbar-expand-lg.navbar-light{:style => 'background-color: #ffad33;'}
+  %a.navbar-brand{:href => '#', :style => 'color: white'} UFoods
+  %span.navbar-toggler-icon
+  #navbarSupportedContent.collapse.navbar-collapse
+    %ul#links.navbar-nav.mr-auto
+      %li.nav-item
+        = link_to 'Home', root_path, class: 'nav-link'
+      - if current_user
+        %li.navbar-text
+          Logged in as: #{current_user.email}
+        %li.nav-item
+          = link_to 'Sign Out', destroy_user_session_path, method: :delete, class: 'nav-link'
```

Comment:
Can you also move the cart info here? 
```
- if @cart
  Cart:
  = @cart.total_unique_items
  - if @cart.shopping_cart_items.any?
    (
    = link_to 'Show cart' , cart_path(@cart, restaurant_id: @restaurant.id)
    )```

And remove the "home" buttons from show-pages both for cart and restaurant pages, since you have it in the navbar :)

Reply:
Yes we can but then we get an error because of the restaurant id, no method error not getting it really

Reply:
On server or in tests @olofvi ?